src/com/javacore/lesson07/FileReadWriter.java

```diff
@@ -0,0 +1,88 @@
+package com.javacore.lesson07;
+
+import java.io.*;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+public class FileReadWriter {
+
+    static final String pathToFile = "src/com/javacore/lesson07/OrdersData.txt";
+
+    public static void readFromFile(ArrayList<Order> orders) throws IOException, InvalidStatusException {
+        int id = 0;
+        OrderStatus statusOrder = null;
+        LocalDateTime startOrderTime = null;
+        LocalDateTime updateOrderTime = null;
+
+        try (BufferedReader bufferReader = new BufferedReader(new FileReader(pathToFile))) {
+
+            String line;
+            StringBuilder stringBuilder;
+            int counter = 1;
+
+            while ((line = bufferReader.readLine()) != null) {
+                stringBuilder = new StringBuilder();
+                for (int i = 0; i < line.length(); i++) {
+                    if (line.charAt(i) == ';') {
```

Comment:
there is an easier way to split string by delimiter https://www.w3schools.com/java/ref_string_split.asp